ical/main.go

```diff
@@ -75,23 +70,23 @@ func (ical *LoadediCal) Modify(e *ics.VEvent) *ics.VEvent {
 	return e
 }
 
-func (iCal *LoadediCal) Filter() {
-	if iCal.isFiltered {
-		log.Log.Warn("Filtering an already filtered calendar: `", iCal.source.Name, "`")
+func (c *LoadediCal) Filter() {
+	if c.isFiltered {
+		log.Logger.Warn("Filtering an already filtered calendar: `", c.source.Name, "`")
```

Comment:
Can you update to structured logging. I think this will cause some `(!BADKEY)` stuff